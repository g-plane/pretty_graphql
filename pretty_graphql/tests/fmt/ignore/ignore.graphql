{
  # pretty-graphql-ignore
  hero {
       name
    height
  }
}

{
  # dprint-ignore
  hero {
       name
    height
  }
}
