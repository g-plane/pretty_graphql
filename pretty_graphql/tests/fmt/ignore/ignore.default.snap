---
source: pretty_graphql/tests/fmt.rs
---
{
  # pretty-graphql-ignore
  hero {
       name
    height
  }
}

{
  # dprint-ignore
  hero {
    name
    height
  }
}
